BootloaderCommonPkg/Library/XhciLib/UsbHcMem.h

```diff
@@ -1,7 +1,7 @@
 /** @file
 Private Header file for Usb Host Controller PEIM
 
-Copyright (c) 2014 - 2017, Intel Corporation. All rights reserved.<BR>
+Copyright (c) 2014, Intel Corporation. All rights reserved.<BR>
 

```

Comment:
This change looks not right, please check other files too.

Reply:
This was from EDK2 without any change.